src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -452,7 +453,7 @@ export default class AircraftController {
             altitude: spawnPatternModel.altitude,
             speed: spawnPatternModel.speed,
             heading: spawnPatternModel.heading,
-            position: spawnPatternModel.position,
+            position: spawnPatternModel.position.toPositionModel(),
```

Comment:
Ah, that makes sense.  In that case, I think this is the correct way to go.  I think we'd run into `super` problems with the current implementation by doing it this way, though.  See comment I left about a helper function to create these new instances.